engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -179,24 +178,12 @@ public boolean pregenerateChunks() {
     @Override
     public void update(float deltaInSeconds) {
 
-        updateTick(deltaInSeconds);
-
-        playerCamera.update(deltaInSeconds);
-        positionShadowMapCamera();
-        shadowMapCamera.update(deltaInSeconds);
-
-        renderableWorld.update();
+        secondsSinceLastFrame += deltaInSeconds;
+        tick += deltaInSeconds * 1000;  // Updates the tick variable that animation is based on.
 
         smoothedPlayerSunlightValue = TeraMath.lerp(smoothedPlayerSunlightValue, getSunlightValue(), deltaInSeconds);
```

Comment:
I don't actually know if smoothedPlayerSunlightValue should be updated on a per tick value. From a renderer perspective I think it could be updated only once per frame. Same as the tick variable actually. I'm wondering if I should put those too in preRenderUpdate();